src/controller/access_control/AccessControl.cpp

```diff
@@ -64,9 +64,11 @@ class AccessControlDelegate : public Access::AccessControl::Delegate
             return CHIP_ERROR_ACCESS_DENIED;
         }
 
-        if (subjectDescriptor.authMode != AuthMode::kCase)
+        // Allow CASE, PASE, and internal device access
+        if (subjectDescriptor.authMode != AuthMode::kCase && subjectDescriptor.authMode != AuthMode::kPase &&
+            subjectDescriptor.authMode != AuthMode::kInternalDeviceAccess)
```

Comment:
Why?  nothing in the group disallows groups here on the access check level.

I really don't understand where all these non-spec-compliant checks are coming from.